### DISCUSSION TITLE
Discussion: can we talk about new edge's navigation view design

### DISCUSSION THREAD

**Discussion by jordan-msftfan** [microsoft/microsoft-ui-xaml#7475] [General]
# Discussion: can we talk about new edge's navigation view

In July 2020, Microsoft discussed the definition of Fluent Design in the Edge discussion forum, and where Fluent Design is headed.It has mention that:

> If you browse through the Fluent guidance at the link above, you’ll note that the new version of Edge is aligned with the current direction Fluent is going, and we will continue to evolve along with the design system and Microsoft as a whole.

In the past discussions, we have created a lot of topics about Acrylic and Reveal, but very little attention has been paid to other parts of the other Edge design area like **Navigation View**.

In an update to the Tips app, there are several screenshots of the new designed Settings app, which also uses the Edge like Navigation View on the left. Microsoft withdrew it within a few days, but it's still worth the attention of WinUI community, but I don't see anyone talking about it in the WinUI Community. (See screenshots below).

I also noticed similar designs in the visual update proposal of the ListView GridView and the Pull Request of the ComboBox.
This is a big difference from the current version of WinUI, and if Edge represents the direction Fluent Design is taking, then the WinUI community should align these specifications.

Please forgive me if I don't speak well, I don't live in an English speaking country.

# Screenshots

![QQ截图20210121102130](https://user-images.githubusercontent.com/77601580/105273183-bfdc9c80-5bd5-11eb-970d-685d01a77223.png)
![Windows-10-VPN-design-mockups](https://user-images.githubusercontent.com/77601580/105274316-e8659600-5bd7-11eb-88d2-291788226fe4.jpg)

# Related Links

Discussion - Updating our interface with Fluent touches [Reply from MissyQ who works in MS](https://techcommunity.microsoft.com/t5/discussions/discussion-updating-our-interface-with-fluent-touches/m-p/1522286/highlight/true#M32775)

Proposal: Update the designs of ListView and GridView #3544 
Combobox visual updates #3938 
Proposal: Consistent Acrylic #3583

**Comment by mdtauk** [comment]
WinUI is moving closer to Edge with the Grid and List View changes being introduced.

The Windows 10X settings app has started moving to the more modern WinUI visual look also, not sure when it will make its way to Windows 10 however.  It is on 10X and Hololens right now.

![image](https://user-images.githubusercontent.com/7389110/105275382-242c3b80-5b97-11eb-8da0-8d910a9dd376.png)

**Comment by jordan-msftfan** [comment]
They may overhaul the Windows 10 Settings App in the rumored "Sun Valley".

Beyond that, it's good to hear that WinUI design specifications are moving closer to Edge.
I haven't seen any news about the change to Navigation View yet (in WinUI3), but I think it will happen in the future.
The current Navigation View is outdated by design, for the most part, it cannot give Windows 10 modern appeal.

However, there is currently not enough information to support the Navigation View will get closer to Edge. For example, in the new Alarms and Clock app UI (#3817 ), Microsoft will still use the existing Navigation View.

So it brings some questions:

1. If the Navigation View gets an update which be closer to the Edge.

- Assume that Acrylic and Reveal have been implemented in WinUI3, whether or not the Navigation View will be **always** used in acrylic like in the current Windows 10 Settings App?

_For the Reveal effect and in proposal Focus Visual, it is easy to think of their actual implementation effect. However, it seems that the Navigation View of edge is not designed for Acrylic at the very beginning._

2. If the navigation view will be improve in other ways.

- Does this introduce inconsistency, assuming that the existing Edge design is the new Fluent specification?
- As i mention before, it cannot give Windows 10 modern appeal.

**Comment by mdtauk** [comment]
I just submitted a proposal for changes to this design, so please feel free to add your thoughts to it, and lets see if Microsoft take any of them on-board when they check in work on this control. #3999
